lib/signed-xml.js

```diff
@@ -612,9 +574,12 @@ SignedXml.prototype.loadReference = function(ref) {
       transforms.push(utils.findAttr(trans, "Algorithm").value)
     }
 
-    var inclusiveNamespaces = xpath.select("//*[local-name(.)='InclusiveNamespaces']", transformsNode);
+    var inclusiveNamespaces = utils.findChilds(trans, "InclusiveNamespaces")
     if (inclusiveNamespaces.length > 0) {
-      inclusiveNamespacesPrefixList = inclusiveNamespaces[0].getAttribute('PrefixList');
+      //Should really only be one prefix list, but maybe there's some circumstances where more than one to lets handle it
+      for (var i = 0; i<inclusiveNamespaces.length; i++) {
+        inclusiveNamespacesPrefixList = inclusiveNamespaces[i].getAttribute('PrefixList');
```

Comment:
Slightly doubtful of this - wouldn't the behaviour change such that you only get the last prefix list? Is that acceptable?

Reply:
Good spot. Fixed with new pull request. Not sure this will ever happen though, but fixed anyway.